### PR TITLE
Fixes #12516: API : status endpoint 

### PR DESCRIPTION
**Pull request by Haypierre**
https://www.rudder-project.org/redmine/issues/12516

### CODE REVIEW

**Comment by Haypierre** [comment]
PR rebased

File: rudder-rest/src/main/scala/com/normation/rudder/rest/lift/SystemApi.scala [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  }
+
+  object Status extends LiftApiModule0 {
+    val schema = API.ApiStatus
+    val restExtractor = restExtractorService
+
+    def process0(version: ApiVersion, path: ApiPath, req: Req, params: DefaultParams, authzToken: AuthzToken): LiftResponse = {
+      implicit val prettify = params.prettify
+      implicit val action = "getStatus"
+      toJsonResponse(None,("gloal" -> "ok"))
```
Comment (VinceMacBuche) [inline rudder-rest/src/main/scala/com/normation/rudder/rest/lift/SystemApi.scala]: global :)

**Comment by VinceMacBuche** [comment]
apart the small typo in the api this is great

**Comment by Haypierre** [comment]
Commit modified

**Pull request merged**